src/algoliasearch.helper.js

```diff
@@ -605,7 +605,7 @@ AlgoliaSearchHelper.prototype._getHitsSearchParams = function() {
     additionalParams.numericFilters = numericFilters;
   }
 
-  return extend(this.state.getQueryParams(), additionalParams);
+  return merge(this.state.getQueryParams(), additionalParams);
```

Comment:
If we are able to write or think about a failing case then ok, right now I cannot think of one